### ISSUE TITLE
Confused about how DRA is implemented

### ISSUE THREAD

**Issue by Skyorca** [liyunsheng13/DRT#11]
Hello, I read your paper and code, but get confused about the implementation of dynamic transfer module.
According to your paper, the model parameters should be adaptive to each sample, but the code below sees that the model parameter is still shared by all samples:

 dyres = self.conv_s1(out)*y[:,0] + self.conv_s2(out)*y[:,1] + \
            self.conv_s3(out)*y[:,2] + self.conv_s4(out)*y[:,3]
  out = dyres + self.conv2(out)

Could you provide some explanations for more details? Thanks a lot.

Best Regards,
Orca

**Comment by liyunsheng13** [comment]
You are correct that all the parameters are shared across samples. But the dynamic coefficient (y in your equation) is dynamic for different samples. It means different samples have different combination from conv_s1 to conv_s4, thereby different residual.
